Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -291,10 +291,11 @@ public string MakeValidUsername(string username)
                     && authenticationProvider != null
                     && !(authenticationProvider is DefaultAuthenticationProvider))
                 {
-                    // We should trust the user that the authprovider says, not what was typed
+                    // trust the username returned by the authentication provider
```

Comment:
Very minor nitpick, but I think most comments start with a capital.

Reply:
Oh, I've been updating comments to lowercase. If you add suggestions to fix this pull request I'll be sure to keep them capitalized from now on.